middleware/faq.rst

```diff
@@ -535,6 +535,251 @@ The resulting query logs can be useful for developers and database administrator
+- A field called ``producer`` connecting each ``datasetType`` for each data ID to the task which produced it
+- Counts of quanta and datasets in all possible states. 
+
+Currently, the ``--force-v2`` option is the suggested usage until version 1 of pipetask report (using the `QuantumGraphExecutionReport` instead of the `QuantumProvenanceGraph`) is deprecated.
+
+With that, we'll go into how to read the output of ``pipetask report``.
+
+The Quanta Table
+----------------
+
+The table for Quanta from the ``w_2024_28`` DC2 test-med-1 reprocessing run, before recoveries, looked like:
+
+.. code:: sh
+
+               Task          Unknown Successful Blocked Failed Wonky TOTAL EXPECTED
+    ---------------------- ------- ---------- ------- ------ ----- ----- --------
+                  makeWarp       0       6596       0      0     0  6596     6596
+     selectDeepCoaddVisits       0        294       0      0     0   294      294
+    selectGoodSeeingVisits       0        294       0      0     0   294      294
+               templateGen       0        288       0      6     0   294      294
+             assembleCoadd       0        280       0     14     0   294      294
+                 detection       0        280      14      0     0   294      294
+    healSparsePropertyMaps       0          1       5      0     0     6        6
+           mergeDetections       0         36      13      0     0    49       49
+                   deblend       0         36      13      0     0    49       49
+                   measure       0        216      78      0     0   294      294
+         mergeMeasurements       0         36      13      0     0    49       49
+           forcedPhotCoadd       0        216      78      0     0   294      294
+          writeObjectTable       0         36      13      0     0    49       49
+      transformObjectTable       0         36      13      0     0    49       49
+    consolidateObjectTable       0          0       1      0     0     1        1
+        matchObjectToTruth       0          0       1      0     0     1        1
+      compareObjectToTruth       0          0       1      0     0     1        1
+
+
+which indicates that there are 6 failed ``templateGen`` quanta, 14 failed ``assembleCoadd`` quanta, 14 blocked ``detection`` quanta, and so on.
+
+These totals are all reached by examining the status of each task run on each data ID from run to run.
+
+Statuses for Quanta 
+^^^^^^^^^^^^^^^^^^^
+
+Unknown
+"""""""
+The ``Unknown`` category could mean one of two things. Being the default value for the status of a task run on a particular data ID, it could technically mean that no attempt has been made to execute this part of the graph. However, in a table that does not look empty, it much more likely denotes that the task's metadata is missing for that data ID. This makes it impossible for us to tell whether the task succeeded or failed, but does point to possible infrastructure problems.
```

Comment:
Standard practice is to write 1 sentence per line. This makes future git history diffs easier to parse if only one sentence is touched in a paragraph. As long as you don't leave an empty line between sentences, they'll still render as a paragraph in the built docs.